demos/DCQC.ipynb

```diff
@@ -133,8 +133,8 @@
    "metadata": {},
    "outputs": [],
```

Comment:
I'm running through the demo, and I'm getting:

```
AttributeError: type object 'TestABC' has no attribute 'list_tests'
```

Reply:
I'm getting the same error, is it because the line: `all_tests = TestABC.list_tests()` should be `all_tests = TestABC.list_subclasses()`?

Reply:
I've decided to delete this file because it duplicates the `test_json_report_generation` test, which passes on my end. I might add it back once I have a more tutorial-like structure that I would like to follow. 

```
tests/test_acceptance.py::test_json_report_generation PASSED
```

Reply:
Makes sense for this ticket. We will need to have a ticket to create examples to enable ease of use for our stakeholders.   Are you thinking of including examples in the sphinx autodocumentation?